gas/testsuite/ChangeLog.COREV

```diff
@@ -1,3 +1,188 @@
+2022-07-29  Charlie Keaney  <charlie.keaney@embecosm.com>
+
```

Comment:
These two lines should be added for each patch in which you're adding a new entry.